server/src/item.rs

```diff
@@ -168,4 +170,41 @@ impl Item {
     pub fn path(&self) -> &OwnedObjectPath {
         &self.path
     }
+
+    pub async fn set_locked(&self, locked: bool) -> Result<(), ServiceError> {
```

Comment:
Why did the method for setting the collection as locked got removed...? The goal of this PR is to just make Lock/Unlock do the locking/unlocking, forget that prompting even exists.

Once that is properly implemented, create a new patch that adds the infrastructure for prompting and update Lock/Unlock to make use of that.

Reply:
Re-added that part with #[allow(unused)]: https://github.com/bilelmoussaoui/oo7/pull/150#discussion_r1858844837